README.md

```diff
@@ -31,46 +45,50 @@ swift package -Xlinker -L/usr/local/lib generate-xcodeproj
 
 #### Linux
 Download the release package for your Linux distribution from http://dev.mysql.com/downloads/repo/apt/
-For example: `wget https://repo.mysql.com//mysql-apt-config_0.8.4-1_all.deb`
+For example: `wget https://repo.mysql.com//mysql-apt-config_0.8.10-1_all.deb`
```

Comment:
@kilnerm does this look good?

Reply:
While we are defaulting to mysql v5.7 it would probably be better to just use apt-get install -y mysql